Makefile

```diff
@@ -71,14 +71,17 @@ lint-container: ## Lint the Symfony DI container
 lint-twig: ## Lint Twig files
 	@bin/console lint:twig templates/
 
+fix-twig: ## Fix Twig files with Twig CS Fixer
```

Comment:
There is some inconsistency between
- lint-php & fix-php
- lint-twig & fix-twig

Lint-php and twig-php are both running PHP-Cs-Fixer
Lint-twig and fix-twig are not running the same tool.

Dunno if an other name should be used for 
`@bin/console lint:twig templates/`...
cause lint-twig could be
`@vendor/bin/twig-cs-fixer lint --fix ./templates`

Reply:
It's a native symfony command, it cannot be modified.

Reply:
I know, I'm talking about the make command you named.

Here you provide twig cs fixer fix command but not the dry run version unlike php cs fixer

Reply:
I agreee `vendor/bin/twig-cs-fixer lint --fix ./templates` should not contain `--fix` for the lint.-, that's what I did in the other PR.